linux/jre/debian/src/main/packaging/temurin/20/debian/rules

```diff
@@ -3,10 +3,10 @@
 pkg_name = temurin-20-jre
 priority = 1912
 jvm_tools = java jfr jrunscript jwebserver keytool rmiregistry
-amd64_tarball_url = https://github.com/adoptium/temurin20-binaries/releases/download/jdk-20.0.1%2B9/OpenJDK20U-jre_x64_linux_hotspot_20.0.1_9.tar.gz
-amd64_checksum = daacf24c15bf7f38a957a98a312911a36ba7f7d97004920a7875791f20e8e1ed
-arm64_tarball_url = https://github.com/adoptium/temurin20-binaries/releases/download/jdk-20.0.1%2B9/OpenJDK20U-jre_aarch64_linux_hotspot_20.0.1_9.tar.gz
-arm64_checksum = 4b04fcfabf833403cc74dd19105a387563f9ff0fef975c4101f3d74c53eb7745
+amd64_tarball_url = https://github.com/adoptium/temurin20-binaries/releases/download/jdk-20.0.1%2B9/OpenJDK20U-jre_x64_linux_hotspot_20.0.2_9.tar.gz
```

Comment:
s|download/jdk-20.0.1|download/jdk-20.-.2| ...

Reply:
I was just fixing that. :)
Good catch Jie.